integration-cli/docker_cli_run_test.go

```diff
@@ -4284,6 +4284,17 @@ func (s *DockerSuite) TestRunCredentialSpecWellFormed(c *check.C) {
 func (s *DockerSuite) TestRunServicingContainer(c *check.C) {
 	testRequires(c, DaemonIsWindows, SameHostDaemon)
 
+	// This functionality does not exist in post-RS3 builds. Note we get the version number
+	// this way as test binaries are not manifested.
+	if runtime.GOOS == "windows" {
```

Comment:
Is this check needed, with the test _requiring_ a Windows daemon _and_ same-host daemon?

Reply:
Fixed